src/main/java/uk/gov/hmcts/reform/et/syaapi/service/ApplicationService.java

```diff
@@ -203,7 +203,7 @@ private JSONObject getDocumentDownload(String authorization, CaseData caseData)
                     downloadDocument.getByteArray(),
                     false,
                     true,
-                    "52 weeks"
+                    "78 weeks"
```

Comment:
Whats this change for? Why is 78 weeks hardcoded

Reply:
NB By default the PDF can remain active in the email for up to 78 weeks. After this time the PDF link should be expired

Reply:
Could the PDF expiry time be put into a constant? or even config?

Reply:
Done